File: Golang/ex1.go
package main

import (
	"fmt"
)

type Carro struct {
	Marca  string
	Modelo string
	Ano    int
}

func main() {

	carro1 := Carro{Marca: "Toyota", Modelo: "Corolla", Ano: 2020}
	carro2 := Carro{Marca: "Ford", Modelo: "Focus", Ano: 2019}
	carro3 := Carro{Marca: "Honda", Modelo: "Civic", Ano: 2021}

	fmt.Println("Detalhes do Carro 1:")
	fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", carro1.Marca, carro1.Modelo, carro1.Ano)

	fmt.Println("Detalhes do Carro 2:")
	fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", carro2.Marca, carro2.Modelo, carro2.Ano)

	fmt.Println("Detalhes do Carro 3:")
	fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", carro3.Marca, carro3.Modelo, carro3.Ano)
}

File: Java/ex1.java
class Carro {
    private String marca;
    private String modelo;
    private int ano;

    public Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    public void exibirDetalhes() {
        System.out.println("Marca: " + this.marca + ", Modelo: " + this.modelo + ", Ano: " + this.ano);
    }

    public static void main(String[] args) {
        
        Carro carro1 = new Carro("Toyota", "Corolla", 2020);
        Carro carro2 = new Carro("Honda", "Civic", 2021);
        Carro carro3 = new Carro("Ford", "Focus", 2019);

        
        carro1.exibirDetalhes();
        carro2.exibirDetalhes();
        carro3.exibirDetalhes();
    }
}
